src/visualizer/graphPanel/GraphPanel.cpp

```diff
@@ -0,0 +1,17 @@
+//
+// Created by stefan on 23.07.2023.
+//
+
+#include "GraphPanel.hpp"
+
+namespace age::visualizer {
+GraphPanel::GraphPanel(QWidget* pParent) noexcept : QWidget(pParent) {}
+
+void GraphPanel::mousePressEvent(QMouseEvent* pEvent) {
+  auto mousePos = pEvent->position();
```

Comment:
To avoid problems with event filtering, call base class function, either before any logic (to use implicit filtering) or after logic (to filter or modify event).
```suggestion
void GraphPanel::mousePressEvent(QMouseEvent* pEvent) {
  GraphPanel::mousePressEvent(pEvent);
  auto mousePos = pEvent->position();
```